docs/book/driver-deployment/installation.md

```diff
@@ -47,6 +47,8 @@ Create a configuration file that will contain details to connect to vSphere.
 
 The default file to write these configuration details is the `csi-vsphere.conf` file. If you would like to use a file with another name, change the environment variable `VSPHERE_CSI_CONFIG` in the deployment YAMLs described in the section [Install vSphere CSI driver](#install) below.
 
+For deployment with zones refer to https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/deploying_csi_with_zones.html
```

Comment:
Will this link be clickable?

Reply:
Yes refer the preview - https://deploy-preview-886--kubernetes-sigs-vsphere-csi-driver.netlify.app/driver-deployment/installation.html#create_csi_vsphereconf